test/powershell/engine/Module/IsolatedModule.Tests.ps1

```diff
@@ -27,10 +27,12 @@ Describe "Isolated module scenario - load the whole module in custom ALC" -Tag '
         $context1.Name | Should -BeExactly "MyCustomALC"
         $context1 | Should -Be $context2
 
-        ## Test-NestedCommand depends on NewtonSoft.Json 10.0.0.0 while PowerShell depends on 13.0.0.0 or higher.
+        ## Test-NestedCommand depends on NewtonSoft.Json 13.0.1.0 while PowerShell depends on 13.0.3.0 or higher.
```

Comment:
The package version is `13.0.1` and `13.0.3`, however, the assembly version is the same from those 2 packages, both are `13.0.0.0`. The purpose of this test is to verify that a lower version assembly can be loaded when powershell already loads a higher version assembly, so the dependency here needs to be lower than `13.0.0.0`.

The dependency here is for a test, and we have dismissed the alert [here](https://github.com/PowerShell/PowerShell/security/dependabot/12) with the appropriate information, so please revert this change.

After Newtonsoft.Json releases a higher assembly version and we've adopted it, I will update the test to use `13.0.0.0`.

Reply:
reverting all of the ALC test changes

Reply:
The version 13.0.3 `NewtonSoft.Json` package still contains the version 13.0.0.0 of the assembly, the same as the 13.0.1 version of the package. So, please revert the changes to this test.

Reply:
this was reverted